api/base/filters.py

```diff
@@ -24,20 +24,22 @@
 def sort_multiple(fields):
     fields = list(fields)
     def sort_fn(a, b):
-        while fields:
-            field = fields.pop(0)
+        sort_direction = 1
+        for index, field in enumerate(fields):
```

Comment:
Gosh, I hate to keep nitpicking ... but we no longer need the enumeration here, since we're no longer using `index` as of be85a83ccbda1f2ee12ea7d42facd0d160a19cab.

I promise we're close, @KAsante95! ☺️